README.md

```diff
@@ -20,8 +20,23 @@ console.log(second_array) // [1, 2, 3]
 
 ```js
 function swap(arr1, arr2) {
-  // -- YOUR ANSWER HERE --
-}
+  
+  let first_arr = [1, 2, 3];
+  let second_array = [4, 5, 6];
+
+
+function swap(arr1, arr2) {
+    arr1Reversed = [ ];
+    arr2Reversed = [ ];
```

Comment:
Are these two variables, `arr1Reveresed` and `arr2Reveresed`, in the global or function scope?